### PR TITLE
feat(mongodb): Make the replica set name configurable

### PR DESCRIPTION
**Pull request by shubhamrajvanshi**
## What does this PR do?
This PR makes the replica set name configurable, instead of using a predefined name rs.

## Why is it important?
A MongoDB replicaset name is a configurable option so I wanted to align test containers with that.

## Related issues
None

-

### CODE REVIEW

**Comment by mdelapenya** [comment]
Hi @shubhamrajvanshi could you take a look at this comment https://github.com/testcontainers/testcontainers-go/pull/2469#discussion_r1593978391? 

We discussed about it when adding the replica set option. I'm not blocking this, but giving you more context. WDYT?

File: modules/mongodb/mongodb.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func WithReplicaSet() testcontainers.CustomizeRequestOption {
 						return err
 					}
 
-					cmd := eval("rs.initiate({ _id: 'rs', members: [ { _id: 0, host: '%s:27017' } ] })", ip)
+					cmd := eval("rs.initiate({ _id: replSetName, members: [ { _id: 0, host: '%s:27017' } ] })", ip)
```
Comment (mdelapenya) [inline modules/mongodb/mongodb.go:95]: Quote replica set name
```suggestion
					cmd := eval("rs.initiate({ _id: '%s', members: [ { _id: 0, host: '%s:27017' } ] })", replSetName, ip)
```

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks!

**Comment by shubhamrajvanshi** [comment]
Thanks @mdelapenya for pushing the relevant changes and getting this merged

**Pull request merged**